src/blocks/vpn.rs

```diff
@@ -0,0 +1,270 @@
+//! Current connection status for vpn networks
+//! This widget toggles the connection on left click.
+//!
```

Comment:
Inserting an empty line after the first line will help `rustdoc` generate a better page at https://greshake.github.io/i3status-rust/i3status_rs/blocks/index.html